src/js/player.js

```diff
@@ -92,6 +92,25 @@ class Player extends Component {
     // see enableTouchActivity in Component
     options.reportTouchActivity = false;
 
+    // If language is not set, get the closest lang attribute
+    if (!options.language) {
+      if (typeof tag.closest === 'function') {
+        let closest = tag.closest('[lang]');
+        if (closest) {
+          options.language = closest.getAttribute('lang');
+        }
+      } else {
+        let element = tag;
+        while (element && element.nodeType === 1) {
```

Comment:
Suggest you use `... && element.nodeType === Node.ELEMENT_NODE` here for clarity (see https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType).